scripts/make_pydocs.sh

```diff
@@ -20,7 +20,7 @@ print_help() {
   echo
   echo "Produce an sphinx docs at pydocs/" 
   echo "Example: ./make_pydocs.sh"
-  exit 1
+  return 1
```

Comment:
Will this change the behavior of the script? I think it may have been intended to early-exit when `print_help` is called.